teps/0091-trusted-resources.md

```diff
@@ -377,13 +386,44 @@ metadata:
   name: feature-flag
   namespace: tekton-pipelines
 data:
-  verification-policy: enforce
+  enable-trusted-resource: true
 ```
 
-`verification-policy`. (Optional, `enforce`, `warn` or `skip`, default `skip`):
- * `enforce`: Failing verification will mark the taskruns/pipelineruns as failed.
- * `warn`: Log warning but don't fail the taskruns/pipelineruns.
- * `skip`: Directly skip the verification.
+`enable-trusted-resource`. (Optional, `true` or `false`, default `false`):
+ * `true`: Failing verification will mark the taskruns/pipelineruns as failed.
+ * `false`: Directly skip the verification.
+
+
+ ```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-trusted-resources
+  namespace: tekton-pipelines
+data:
+  no-match-policy: "deny"
+```
+
+`no-match-policy`. (Optional, `allow`, `deny` or `warn`, default `deny`):
+ * `deny`: when no matching policies found, reject the resource and fail taskrun/pipelinerun.
+ * `allow`: when no matching policies found, allow the resource.
+ * `warn`: when no matching policies found, allow the resource and log warning.
+
+ **Note:** If users want to have both signed resources and unsigned resources. The `enable-trusted-resource` needs to be set to `true` and `no-match-policy` is set to `allow`. So for signed resources, matched policies can be used to verify. For unsigned resources, no policies will be found so they are allowed.
```

Comment:
This isn't quite true, since you could also have warn. What might be more useful here are some examples of how the policies + flag interact with each other (this would also be a good chance to document how multiple matching policies interact)